compute_tools/src/http/api.rs

```diff
@@ -165,6 +167,46 @@ async fn routes(req: Request<Body>, compute: &Arc<ComputeNode>) -> Response<Body
             }
         }
 
+        (&Method::POST, "/grants") => {
+            info!("serving /grants POST request");
```

Comment:
Similar debug comment to the other PR

Reply:
As discussed in the other thread, this will remain info level for now.